eng/tools/npm-helpers.ps1

```diff
@@ -0,0 +1,76 @@
+$EngPath = Resolve-Path "${PSScriptRoot}/.."
+$Engcommon = Join-Path $EngPath "common"
+$EngCommonScriptsPath = Join-Path $Engcommon "scripts"
+. (Join-Path $EngCommonScriptsPath common.ps1)
+
+function Get-PackageProperties($pkgArtifact, $workingDirectory)
+{
+  if ($GetPackageInfoFromPackageFileFn -and (Test-Path "Function:$GetPackageInfoFromPackageFileFn"))
```

Comment:
Given you are using this only for js and you are in the js repo you can directly call the js version of this function.